src/aer-types/types/data.ts

```diff
@@ -90,6 +90,12 @@ export type Expansions = {
   [id: string]: Expansion
 }
 
+export function formatExpansionName(expansion: Expansion): string {
```

Comment:
could you please move this out of the types package. A better place for this function would be `src/helpers/index.ts` :)